### PR TITLE
[TASK] Overhaul appendix page

### PR DESCRIPTION
**Pull request by linawolf**
* Use confvals
* Correct indentation

releases: main, 12.4, 11.5

### CODE REVIEW

**Review by brotkrueml — COMMENTED** [review]
Actually, I don't understand what this section is about? The included script is part of the ContentObjectRenderer?

When I guess, I assume, this is about a USER function? But then the heading is totally misleading.

**Comment by linawolf** [comment]
I agree the page needs a overhault, but not in this ticket...

File: Documentation/AppendixA/Index.rst [hunk 1]
```diff
@@ -65,45 +65,48 @@ Call the function :php:`$GLOBALS['TSFE']->set_no_cache()`, if you want to
 disable caching of the page. Call this during development only! And
 call it, if the content you create may not be cached.
 
-**Note:** If you make a syntax error in your script that keeps PHP
-from executing it, then the :php:`$GLOBALS['TSFE']->set_no_cache()`
-function is not executed and the page *is* cached! So in these
-situations, correct the error, clear the page-cache and try again.
-This is true only for :typoscript:`USER` and not for :typoscript:`USER_INT`, which is
-rendered *after* the cached page!
+.. note::
```
Comment (brotkrueml) [inline Documentation/AppendixA/Index.rst:68]:
```suggestion
..  note::
```

**Review by brotkrueml — APPROVED** [review]
Well, you named it "Overhaul" - that includes also the content IMHO ;-)
You sure, you want to do this? You'll have fun while backporting - and it's not the work worth IMHO.

**Pull request merged**